secret/store_test.go

```diff
@@ -95,6 +97,14 @@ const (
 	gkeClientCert   = "client_x509_cert_url"
 )
 
+const (
```

Comment:
I think you can reuse the constants from the package, no need to repeat them here.